packages/manager/apps/container/src/container/common/language/index.tsx

```diff
@@ -22,6 +24,9 @@ function LanguageMenu({
   const shell = useShell();
   const [show, setShow] = useState(false);
   const handleRootClose = () => setShow(false);
+  const isSmallDevice = useMediaQuery({
```

Comment:
Maybe we can also done the same things using only CSS and 2 span for language label.